### PR TITLE
Adding a User Profile panel

### PR DESCRIPTION
**Pull request by omgkaustav**

### CODE REVIEW

File: src/components/settings/Profile.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import { Descriptions } from 'antd'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components/macro'
+
+import { UserAvatar } from 'components/shared'
+import { selectUserProfile } from 'store/userSlice'
+
+const Profile = () => {
+    const profile = useSelector(selectUserProfile)
+    console.log(profile)
```
Comment (relaxxpls) [inline src/components/settings/Profile.jsx:11]: Remove all console functions (log, error, etc)

File: src/components/settings/SettingsContainer.jsx [hunk 2]
```diff
@@ -78,7 +80,8 @@ const SettingsContainer = () => {
           </Typography.Link>
         }
       >
-        <Empty description={<PageSubtitle>Coming soon!</PageSubtitle>} />
+        {/* <Empty description={<PageSubtitle>Coming soon!</PageSubtitle>} /> */}
```
Comment (relaxxpls) [inline src/components/settings/SettingsContainer.jsx:83]: You can remove this line

File: src/components/settings/Profile.jsx (hunk 1 above)
Comment (relaxxpls) [inline src/components/settings/Profile.jsx]: Remove all console functions (log, error, ...etc)

**Comment by omgkaustav** [comment]
Removed console functions

**Review by relaxxpls — APPROVED** [review]

**Pull request merged**